packages/web-components/components/selfie/src/SelfieCaptureScreens.js

```diff
@@ -7,12 +7,25 @@ import packageJson from '../../../package.json';
 
 const COMPONENTS_VERSION = packageJson.version;
 
+const smartCameraWeb = document.querySelector('smart-camera-web');
```

Comment:
Same with this and the other web components. From your tests, did document.querySelector return a valid document?

Reply:
Yeah, it worked in both new and legacy components. But I've added the optional chaining as an extra safety net.